libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccountManager.java

```diff
@@ -263,13 +263,32 @@ public void switchToNewUser() {
 		final Bundle reply = new Bundle();
 		final Intent i = new Intent(context, SalesforceSDKManager.getInstance().getLoginActivityClass());
 		i.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-        final Bundle options = SalesforceSDKManager.getInstance().getLoginOptions().asBundle();
-        i.putExtras(options);
+		final Bundle options = SalesforceSDKManager.getInstance().getLoginOptions().asBundle();
+		i.putExtras(options);
         reply.putParcelable(AccountManager.KEY_INTENT, i);
 		context.startActivity(i);
 	}
 
 	/**
+	 * Kicks off the login flow to switch to a new user with jwt. Once the login
+	 * flow is complete, the context will automatically become the
+	 * new user's context and a call to peekRestClient() or getRestClient()
+	 * in ClientManager will return a RestClient instance for the new user.
+	 *
+	 * @param jwt JWT.
+	 * @param url Instance/My domain URL.
+	 */
+	public void switchToNewUser(String jwt, String url) {
+		final Bundle reply = new Bundle();
```

Comment:
@helenren A lot of the code here is the same as the previous method, except for one line that builds ```options```. Let's create one common ```private``` method that does all the common stuff that takes in a ```Bundle``` - ```options```.